### PR TITLE
T&R ReputationView

### PR DESCRIPTION
**Pull request by pimms**
# Why?

The second view on the T&R chopping block, the `ReputationView` is recreated with its new design in FinniversKit.

# What?

Similarly to `IdentityView`, the `ReputationView` must be able to adaptively receive a `ReputationViewModel` after the initial creation. The `Tweakable` options in the demo demonstrates this feature.

# Show me

### Undefined view-model
Screen Shot 2019-09-13 at 10 31 16

### Assigned view-model
Screen Shot 2019-09-13 at 10 31 10
Screen Shot 2019-09-13 at 10 31 13

### CODE REVIEW

File: Demo/Demo/Demo.swift [hunk 1]
```diff
@@ -66,9 +66,7 @@ public enum ComponentViews: String, CaseIterable {
     case tweakable
     case saveSearchView
     case identityView
-    case stepSlider
-    case verificationView
-    case panel
```
Comment (pimms) [inline Demo/Demo/Demo.swift]: Ops, will fix!

**Comment by fespinoza** [comment]
uh, when merging master into a branch i was working on, the test was failing for a missing `reputationView` component test.

It's weird that the CI didn't pick this up @_@

**Comment by bstien** [comment]
Yeah, [we've had this issue before](https://github.com/finn-no/ios-handbook/blob/master/meeting/19-03-07.md#circleci-and-finniverskit-snapshot-tests) with the snapshot tests😕 Not sure what to do about it, though.

**Comment by ag12** [comment]
Strange.

**Comment by pimms** [comment]
Oh snap, I forgot about that. New PR incoming now.

**Comment by pimms** [comment]
Great! Thanks a bunch 🙂

**Pull request merged**